waspc/src/Wasp/Generator/SdkGenerator.hs

```diff
@@ -188,13 +188,9 @@ npmDepsForSdk spec =
             ("mitt", "3.0.0"),
             ("react", "^18.2.0"),
             ("lodash.merge", "^4.6.2"),
-            ("react-router-dom", "^5.3.3"),
+            ("react-router-dom", "^6.26.1"),
             ("react-hook-form", "^7.45.4"),
-            ("superjson", "^1.12.2"),
-            -- Todo: why is this in dependencies, should it be in dev dependencies?
-            -- Should it go into their package.json
-            ("@types/express-serve-static-core", "^4.17.13"),
-            ("@types/react-router-dom", "^5.3.3")
```

Comment:
React Router 6 comes with types, no need for the `@types/` package.

Reply:
It's weird that's part of the dependencies (and not dev dependencies), right? Or am I missing something?

Reply:
My bad, brain fart.